sdks/cpp/lite/include/Param.h

```diff
@@ -14,20 +15,19 @@
 namespace catena {
 namespace lite {
 
-
-
 /**
  * @brief Param provides convenient access to parameter values
  * @tparam T the parameter's value type
  */
-template <typename T> class Param : public IParam {
-  public:
+template <typename T> 
+class Param : public IParam {
+public:
     /**
      * @brief OidAliases is a vector of strings
      */
     using OidAliases = std::vector<std::string>;
```

Comment:
consider adding a `using IConstraint = common::IConstraint;` to simplify line 65.